<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsExplorer</class>
 <widget class="QWidget" name="SettingsExplorer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>547</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="aircraft_combo"/>
     </item>
     <item>
      <widget class="QPushButton" name="home_settings">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../resources/ressources.qrc">
         <normaloff>:/pictures/settings.svg</normaloff>:/pictures/settings.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="path_layout"/>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QSplitter" name="splitter">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QListWidget" name="menu_list"/>
        <widget class="QScrollArea" name="scrollArea_2">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="settings_area">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>68</width>
            <height>209</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2"/>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../resources/ressources.qrc"/>
 </resources>
 <connections/>
</ui>
